README.md

```diff
@@ -269,6 +272,44 @@ The Kubernetes system is trying to scale a StatefulSet named fake-deployment usi
 
 </details>
 
+_Manage integrations_
```

Comment:
Can we add the whole block in details? I think we're getting to a point where ALL of these commands should be in docs.k8sgpt.ai only as the README is getting very long, what do you think?

Reply:
I agree